src/core/database/index.js

```diff
@@ -9,7 +9,10 @@ const config = knexConfig[environment];
 const connection = knex(config);
 export default connection;
 
-export const getTransaction = () => connection.transaction();
+export const getTransaction = async () => {
+    const trx = await connection.transaction();
+    return trx;
+};
```

Comment:
You can just return a promise message without await. I think the old one is easy enough to understand.